doc/guides/getting_started/src/e2e_regression_setup.md

```diff
@@ -0,0 +1,58 @@
+ - **Device test** software, it is a C program built by bazel that run in the flash after the ROM and perform checking in the HW and/or memories to check the configurations performed by the ROM.
+ - **FPGA bitstream**, it is built by bazel which using vivado.
+ - **ROM Image**, it is a C program built by bazel which is the DUT (Device Under Test).
+ - **OTP image**, contains secure configurations read by the ROM to define workflows such as the lifecycle stage, so for some tests may be executed with several different OTP images to test all corner cases.
+  
+![Architecture](ROM_E2E_Tests.svg)
+
+All the existing tests are documented in the [ROM E2E Testplan](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson).
+## Hardware setup
+### FPGA
+The E2E ROM tests run in the CW310 FPGA board with the following configuration:
+ - The power supply connected to the J11 connector.
+ - A USB cable connecting the Host PC to the USB-C Data (J8) connector. 
+ - The USRDIP#0 (SW4 header) is set to 0.
+ - The rest of the USRDIPs are set to 1.
+ - The S3 switch (next to USB connector J9) is set to USB.
+ - The S2 switch (next to JTAG header) is set to +3.3V.
+ - Optionally, a USB cable connecting the host PC to the USB-C J9 connector for future tests that will test the USB block.
+  
+### JTAG adapter
+Some tests will use the JTAG interface to interact with Opentitan, as the CW310 doesn't have a embedded JTAG adapter, we can use an external adapter (Olimex ARM-USB-TINY-H JTAG) connected to the J13 header in the CW310.
+
+### Full hardware setup
+![Full hardware setup](CW310_setup.png)
+
+## Software setup 
+The software dependencies are covered by other guides in this book and are linked below.
+- Clone the Opentitan repository and install the software dependencies: [Get started](README.md).
+- Setup the FPGA: [FPGA guide](./setup_fpga.md).
+- Install Vivado to be able to splice bitstreams: [Installing Vivado](install_vivado/README.md).
+
+**Note**: Make sure that you added the udev rules for the [FPGA](install_vivado/README.md#device-permissions-udev-rules) board and the [JTAG](setup_fpga.md#device-permissions-udev-rules) adapter.
+
+## Running the ROM E2E tests
+The ROM E2E tests are listed in the bazel [BUILD file](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/rom/e2e/BUILD), alternatively they can be listed with the command:
+```sh
+bazel query 'attr(tags, cw310, tests(//sw/device/silicon_creator/rom/e2e/...))
+``` 
+Once the hardware setup is done and the FPGA is connected to the USB, the ROM E2E tests can be executed with the command `bazel test` pointing at the `sw/device/silicon_creator/rom/e2e` directory.
+```sh
+cd ${REPO_TOP}
+bazel test --define DISABLE_VERILATOR_BUILD=true --define bitstream=vivado --test_tag_filters=-verilator,-dv,-broken --build_tests_only //sw/device/silicon_creator/rom/e2e/...
```

Comment:
I'm a bit loathe to tell people to use `bitstream=vivado`.  That causes bazel to invoke vivado and synthesize the bitstream.  If you're modifying the RTL, then this is the right thing to do.  However, if you're only modifying the software, then `bitstream=gcp_splice` is more likely what you wwant.

Reply:
I changed the text to make it clear that the `gcp_splice` is the desirable way, but `vivado` is also possible.